src/golang/lib/repos/tests/require.go

```diff
@@ -193,3 +194,30 @@ func requireDeepEqualOperators(t *testing.T, expected, actual []models.Operator)
 		requireDeepEqual(t, expectedOperator, foundOperator)
 	}
 }
+
+// requireDeepEqualLoadOperators asserts that the expected and actual lists of
+// LoadOperators contain the same elements.
+func requireDeepEqualLoadOperators(t *testing.T, expected, actual []views.LoadOperator) {
+	require.Equal(t, len(expected), len(actual))
+
+	for _, expectedOperator := range expected {
+		found := false
+		var foundOperator views.LoadOperator
+
+		for _, actualOperator := range actual {
+			if (expectedOperator.OperatorName == actualOperator.OperatorName &&
```

Comment:
can this explicit checking be replaced with a call to `requireDeepEqual`?

Reply:
Good suggestion! Updated requireDeepEqualDAGEdges and requireDeepEqualLoadOperators to check with reflect.DeepEqual